.dadew

```diff
@@ -7,6 +7,7 @@
         "cacert",
         "curl-7.73.0",
         "java-openjdk-8u201",
+        "jq",
```

Comment:
we have jq as part of msys already. Can we use that instead of installing it twice?

Reply:
Oh, missed that. I'll try to re-use it.

Reply:
Done.